Sources/Foundation/Host.swift

```diff
@@ -165,6 +165,8 @@ open class Host: NSObject {
 
           pAdapter = pAdapter!.pointee.Next
         }
+        // The force-uwrap here is safe, as if _type is .current then _info is always non-nil.
+        _names = [_info!]
```

Comment:
You could change the signature of this function to `_resolveCurrent(withInfo info: String)` and fixup the callers since `_info` is being unwrapped in the `if` statements. Would simplify having to reason about the force unwrap.

Reply:
Sure, sounds good.